### PR TITLE
[Issue #5] New import

### PR DESCRIPTION
**Pull request by amoose**
Imports data from NCTD (North County Transit District) directly from their GTFS feed
Removes links to sign in/up forms
Provides basic visibility of trains departing today by station

The `db:seed` rake task is triggered with `rake db:seed` or when resetting the database via `rake db:reset` and setup via `rake db:setup`

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,62 @@
...
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require File.expand_path('../../lib/seed_helper.rb', __FILE__)
+require 'csv'
+require 'open-uri'
+require 'colorize'
+
```
Comment (trevormast) [inline db/seeds.rb]: This looks awesome!  I think we should add steps to clear the database first though, as after running this I now have 1022 trains in my dev db. 😅

**Comment by trevormast** [comment]
Got a few of these errors after rake db:seed

```x ERROR! argument out of range 
x #<GTFS::StopTime:0x007f9a2fa32340 @trip_id="11383783", @arrival_time="24:04:00", @departure_time="24:04:00", @stop_id="28002", @stop_sequence="6", @stop_headsign=nil, @pickup_type=nil, @drop_off_type=nil, @shape_dist_traveled="34.0996212", @timepoint=nil> ```

File: db/seeds.rb (hunk 1 above)
Comment (amoose) [inline db/seeds.rb]: seeds.rb is intended to seed the database only; use `rake db:reset` to reset the db and seed.

Comment (amoose) [inline db/seeds.rb]: you are correct, though, that we need to add a rake task to destroy existing trains when it is time to refresh all of the train data, but not the user data.

**Comment by amoose** [comment]
Yes, I also receive those errors; because the time values are greater than 24. I am still investigating whether or not we need to manually massage those entries or ignore.

File: app/models/train.rb [hunk 2]
```diff
@@ -30,33 +34,32 @@ def next_departure(date=Date.today)
...
-		else
-			 time.strftime("%I:%M%p")
-		end
+  def formatted_time
+    departure_time.strftime('%r')
+		# if time_today.strftime('%I').to_i < 10
+		# 	"#{time_today.strftime('%I').to_s[-1..-1]}#{time_today.strftime(':%M%p')}"
+		# else
+		# 	 time_today.strftime("%I:%M%p")
+		# end
```
Comment (amoose) [inline app/models/train.rb]: need to kill these lines 🔥

File: config/secrets.yml.example [hunk 3]
```diff
@@ -0,0 +1,8 @@
+development:
+  google_maps_api_key: YOUR GOOGLE API KEY
+  secret_token: YOUR SESSION SECRET
+
+production:
+
+test:
+  secret_token: '4038e98c8b1c1b101fc1cc78c7e14f9f7257517052fca5c1953d986d455ffdc0e5a313eb947da45ccc98fd1e75a1a8cebee4c69c3e5368ceb727d48d3592bf7f'
```
Comment (amoose) [inline config/secrets.yml.example]: This file needs to be removed 🔥

File: db/seeds.rb (hunk 1 above)
Comment (amoose) [inline db/seeds.rb]: seeds.rb does need to be idempotent. good call.

**Pull request merged**